data-raw/simulate_multiple_phenotypes.R

```diff
@@ -0,0 +1,101 @@
+set.seed(982348)
+library(dplyr)
+library('MASS')
+
+load('data/random_genotype_matrix.rda')
+
+X <- random_genotype_matrix
+X_mean <- apply(X, 2, mean)
+X_sd <- apply(X, 2, sd)
+X <- ((t(X) - X_mean) / X_sd)
```

Comment:
I imagine there is probably a centering and scaling function for matrices.

Just found one:

https://stat.ethz.ch/R-manual/R-patched/library/base/html/scale.html